lib/sql_string.js

```diff
@@ -130,11 +122,11 @@ function zeroPad(number, length) {
 }
 
 function convertTimezone(tz) {
-  if (tz == "Z") return 0;
+  if (tz === "Z") return 0;
 
   var m = tz.match(/([\+\-\s])(\d\d):?(\d\d)?/);
   if (m) {
-    return (m[1] == '-' ? -1 : 1) * (parseInt(m[2], 10) + ((m[3] ? parseInt(m[3], 10) : 0) / 60)) * 60;
+    return (m[1] === '-' ? -1 : 1) * (parseInt(m[2], 10) * 60 + (m[3] ? parseInt(m[3], 10) : 0));
   }
-  return false;
+  return 0;
```

Comment:
I am agree with throwing an error is useful for decreasing mistakes of timezone format.
Sorry, I don't know about the reason for returning 0 in case tz is "Z".

On the other hand, node-mysql2 need to have a compatibility with node-mysql.
From the node-mysql source code, "return false" would be better for consistency.
https://github.com/felixge/node-mysql/blob/master/lib/protocol/SqlString.js#L177

Thus, in this PR, I would keep original "return false".
I guess that it would be better to "throwing an error", after node-mysql change.